src/jwt/jwt.ts

```diff
@@ -134,7 +136,7 @@ export const doOffline = (key: string, val: string, configSsoUrl?: string) => {
       scopes.push(partnerScope);
     }
 
-    if (ssoScopes) {
+    if (ssoScopes && !itLessKeycloakEnv) {
```

Comment:
Just a double-check. The offline token is not supposed to have all SSO scopes of the current session?